packages/components/pie-cookie-banner/package.json

```diff
@@ -19,6 +20,7 @@
     "lint:style:fix": "yarn lint:style --fix",
     "watch": "run -T vite build --watch",
     "test": "echo \"Error: no test specified\" && exit 0",
+    "test:unit": "run -T vitest run test/unit",
```

Comment:
Added unit tests, but how do we ensure they run in CI? @siggerzz

Reply:
@fernandofranca Can you remove `test:unit` and modify this to `"test": "run -T vitest run test/unit"`. This should fix the issue :)